gwlfe/gwlfe.py

```diff
@@ -16,7 +16,7 @@
 from . import WriteOutputFiles
 
 
-def run():
+def run(z):
```

Comment:
Yeah, I don't have strong opinion on this. It was more or less chosen at random. I wouldn't mind if somebody changed this as long as we use the same naming convention throughout.